src/mod.ts

```diff
@@ -0,0 +1,20 @@
+export * from "./types/mod.ts";
+export * from "./util.ts";
+
+export * from "./array/mod.ts";
+export * as Array from "./array/mod.ts";
+
+export * from "./bitflags/mod.ts";
+export * as Bitflags from "./bitflags/mod.ts";
+
+export * from "./primitives/mod.ts";
+export * as Primitives from "./primitives/mod.ts";
+
+export * from "./string/mod.ts";
+export * as String from "./string/mod.ts";
```

Comment:
While I get the need for namespacing and this very much makes sense I am afraid it will clash with the built in `String` and `Array` constructors which isn't very nice. Although I have no better suggestion... Perhaps making the namespaces lowercase?

Reply:
maybe a idea to make it `Strings` and `Arrays` like `Primitives` and `Bitflags`

Reply:
Yeah sure, I am positive to that.